hack/deploy-hub-local/lab-sushy.sh

```diff
@@ -3,16 +3,16 @@
 PS_SUSHY=$(ps -ef | grep py | grep sushy)
 if [ -z "$PS_SUSHY" ]; then
   cp ./lab-sushy.conf /etc/sushy.conf
-  cp ./lab-sushy.service /etc/systemd/system/sushy-emulator.service
+  cp ./lab-sushy.service /etc/systemd/system/sushy.service
 
   dnf -y install pkgconf-pkg-config libvirt-devel gcc python3-libvirt python3 git python3-netifaces
-
+  pip3 install sushy-tools --ignore-installed PyYAML || true
   systemctl daemon-reload
-  systemctl enable --now sushy-emulator.service
+  systemctl enable --now sushy.service
 
   firewall-cmd --zone=libvirt --permanent --add-port=8000/tcp
   firewall-cmd --reload
-  systemctl start sushy-emulator.service
+  systemctl start sushy.service
```

Comment:
This is not necessary with the `systemctl enable --now sushy.service` we have in line 11

Reply:
if you restart firewalld you need to restart also sushy, at least in my tests